spec/repositories/manual_repository_spec.rb

```diff
@@ -131,10 +146,25 @@
       expect(manual_record).to have_received(:save!)
     end
 
-    it "calls dump on the section association marshaller with the manual domain object and edition" do
+    it "saves associated sections and removed sections" do
+      repo.store(manual)
+
+      expect(section_repository).to have_received(:store).with(section)
+      expect(section_repository).to have_received(:store).
+        with(removed_section)
+    end
+
+    it "updates associated document ids on the edition" do
+      repo.store(manual)
+
+      expect(edition).to have_received(:section_ids=).with([section.id])
+    end
+
+    it "updates associated removed document ids on the edition" do
```

Comment:
Minor: "associated removed document ids" -> "associated removed section ids" in spec name.